lib/FrameManager.js

```diff
@@ -87,6 +87,14 @@ class FrameManager extends EventEmitter {
     return Array.from(this._frames.values());
   }
 
+  /**
+   * @param {!string} frameId
+   * @return {!Frame}
+   */
+  frame(frameId) {
+    return this._frames.get(frameId);
```

Comment:
```js
 return this._frames.get(frameId) || null;
```

Reply:
Thank you very much!
https://github.com/GoogleChrome/puppeteer/pull/1646/commits/6a3de8f93550fa179efc10cf58458981f9c66c36